lms/djangoapps/grades/tests/test_new.py

```diff
@@ -168,18 +168,18 @@ def test_create_internals(self):
             with patch(
                 'lms.djangoapps.grades.new.subsection_grade.SubsectionGradeFactory._get_bulk_cached_grade',
                 wraps=self.subsection_grade_factory._get_bulk_cached_grade
-            ) as mock_get_saved_grade:
+            ) as mock_get_persisted:
```

Comment:
Revert this change so it's not confused with the new "get_persisted" method.  Also, consider renaming it to `mock_bulk_cached_grade`.